x-pack/metricbeat/module/aws/ec2/_meta/data.json

```diff
@@ -95,6 +95,33 @@
         "duration": 115000,
         "module": "aws"
     },
+    "host": {
+        "cpu": {
+            "pct": 1.166484417714406
+        },
+        "disk": {
+            "read": {
+                "bytes": 0
+            }
+        },
+        "diskio": {
+            "write": {
+                "bytes": 0
+            }
+        },
+        "id": "i-0516ddaca5c1d231f",
+        "name": "mysql-test",
+        "network": {
+            "in": {
+                "bytes": 6644.4,
+                "packets": 44.6
```

Comment:
Is this a counter? Why is it a decimal number?

Reply:
It's a gauge actually and it's an average value among 5 minutes:

<img width="1374" alt="Screen Shot 2020-07-31 at 8 36 46 AM" src="https://user-images.githubusercontent.com/14081635/89046459-36c16880-d30a-11ea-8a15-fc45d112e355.png">